docs/ibis-for-dplyr-users.ipynb

```diff
@@ -0,0 +1,777 @@
+{
```

Comment:
<p>Interesting, didn't realize this.</p><p> </p><p>We could potentially support this but it might get pretty hairy. What does dplyr do when there's ambiguity, e.g., <code>mutate(x=x + 1, y=x + 1)</code>? Which <code>x</code> gets used there?   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/6050/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>I didn't actually know. But:</p><p> </p><pre class="ql-syntax" spellcheck="false">&gt; data.frame(x=1:3) |&gt;&nbsp;
&nbsp; mutate(x = x + 1, y = x + 1)
&nbsp; x y
1 2 3
2 3 4
3 4 5
</pre>

Reply:
Cool, I wonder if the order of the usage matters but we can leave that for later!